lib/sorcery/controller/config.rb

```diff
@@ -12,6 +12,8 @@ class << self
         attr_accessor :save_return_to_url
         # set domain option for cookies
         attr_accessor :cookie_domain
+        # Set token randomness
+        attr_accessor :token_randomness
```

Comment:
💭  IMO, it seems that `Sorcery::Model::Config` is singleton so it is not bad to access the instance as same as `Sorcery::Controller::Config`